compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/test/concretize_mma_shapes.mlir

```diff
@@ -33,6 +33,38 @@ func.func @concretize_multi_mma_F32_16x16x16_F16(%lhs: tensor<2x2x16x16xf16>, %r
 
```

Comment:
Can you add tests for the new intrinsics to `distribute_mma_to_lanes.mlir` as well, that's the main tests for the layouts.